larky/src/main/java/com/verygood/security/larky/modules/VaultModule.java

```diff
@@ -188,17 +194,90 @@ public void delete(Object value, Object storage) throws EvalException {
         vault.delete(value, storage);
     }
 
+    @StarlarkMethod(
+        name = "sign",
+        doc = "signs a payload with given keyId",
+        parameters = {
+            @Param(
+                name = "keyId",
+                doc = "key ARN to sign with",
+                allowedTypes = {
+                    @ParamType(type = String.class)
+                }
+            ),
+            @Param(
+                name = "message",
+                doc = "the payload to sign",
+                allowedTypes = {
+                    @ParamType(type = String.class)
+                }
+            ),
+            @Param(
+                name = "algorithm",
+                doc = "the algorith to use to sign.",
```

Comment:
do we need to include the algorithms here or make this an enum? i cannot guess what values go here so i need docs.

Reply:
i'm fine adding the choices in the doc.